tests/DfE.FindInformationAcademiesTrusts.Data.AcademiesDb.UnitTests/Extensions/StringExtensionsTests.cs

```diff
@@ -89,4 +89,19 @@ public void ParseAsNullableInt_should_return_correctly_parsed_int(string? input,
         var result = input.ParseAsNullableInt();
         result.Should().Be(expected);
     }
+
+    [Theory]
+    [InlineData("0", 0D)]
+    [InlineData("1", 1D)]
+    [InlineData("1.5", 1.5)]
+    [InlineData("1000", 1000D)]
+    [InlineData("99.99", 99.99)]
+    [InlineData("0.01", 0.01)]
+    [InlineData("", null)]
+    [InlineData("test", null)]
+    public void ParseAsNullableDouble_should_return_correctly_parsed_int(string? input, double? expected)
+    {
+        var result = input.ParseAsNullableDouble();
+        result.Should().BeApproximately(expected, 0.01F);
```

Comment:
It's a bit odd to specify the `0.01` as a float when we're dealing with doubles
```suggestion
        result.Should().BeApproximately(expected, 0.01);
```
By default I think a decimal number here will be a double, if removing the `F` doesn't work then pop a `d` on the end